providers/aws-ec2/src/test/java/org/jclouds/aws/ec2/compute/AWSEC2ComputeServiceLiveTest.java

```diff
@@ -173,15 +170,6 @@ public void testExtendedOptionsAndLogin() throws Exception {
             monitoringApi.close();
          }
 
-         // make sure we made our dummy group and also let in the user's group
-         assertEquals(newTreeSet(instance.getGroupNames()), ImmutableSortedSet.<String> of("jclouds#" + group, group));
-
-         // make sure our dummy group has no rules
-         SecurityGroup secgroup = getOnlyElement(securityGroupApi.describeSecurityGroupsInRegion(instance
-                  .getRegion(), "jclouds#" + group));
-
-         assert secgroup.size() == 0 : secgroup;
```

Comment:
Note: This is not arbitrary. This should have been removed in https://github.com/jclouds/jclouds/pull/947